modules/distr-tracing-tempo-config-query-frontend.adoc

```diff
@@ -8,60 +8,185 @@
+|type: object, additionalProperties: type: string
+|
+
+|`replicas` (inside component)
+|Replicas represents the number of replicas to create for this component.
+|type: integer, format: int32
+|
+
+|`tolerations` (inside component)
+|Tolerations defines component specific pod tolerations.
+|type: array, x-kubernetes-list-type: atomic
+|
+
+|`Tolerations` (items, inside component)
+|The pod this Toleration is attached to tolerates any taint that matches the triple <key, value, effect> using the matching operator <operator>.
+|type: object
 |
+
+|`effect` (inside component.tolerations[x])
+|Effect indicates the taint effect to match. Empty means match all taint effects. When specified, allowed values are NoSchedule, PreferNoSchedule, and NoExecute.
+|type: string
+|
+
+|`key` (inside component)
+|Key is the taint key that the toleration applies to. Empty means match all taint keys. If the key is empty, operator must be Exists; this combination means to match all values and all keys.
+|type: string
+|
+
+|`operator` (inside component)
+|Operator represents a key's relationship to the value. Valid operators are Exists and Equal. Defaults to Equal. Exists is equivalent to wildcard for value, so that a pod can tolerate all taints of a particular category.
+|type: string
+|
+
+|`tolerationSeconds` (inside component)
+|TolerationSeconds represents the period of time the toleration (which must be of effect NoExecute, otherwise this field is ignored) tolerates the taint. By default, it is not set, which means tolerate the taint forever (do not evict). Zero and negative values will be treated as 0 (evict immediately) by the system.
+|type: integer, format: int64
 |
 
-|options:
-  log-level:
-|Logging level for Query.
-|`debug`, `info`, `warn`, `error`, `fatal`, `panic`
+|`value` (inside component)
+|Value is the taint value the toleration matches to. If the operator is Exists, the value should be empty, otherwise just a regular string.
+|type: string
 |
 
-|options:
-  query:
-    base-path:
-|You can set the base path for all tempo-query HTTP routes to a non-root value: for example, `/tempo` will cause all UI URLs to start with `/tempo`. This can be useful when running `tempo-query` behind a reverse proxy.
-|`/<path>`
+|`jaegerQuery`
+|JaegerQuerySpec defines Jaeger Query specific options.
 |
+|
+
+|`enabled`
+|When `enabled`, creates the Jaeger Query component (`jaegerQuery`).
+|type: boolean
+|
+
+|`ingress`
+|Ingress defines Jaeger Query Ingress options.
+|type: object
+|
+
+|`annotations`
+|Annotations defines the annotations of the Ingress object.
+|type: object, additionalProperties: type: string
+|
+
+|`host`
+|Host defines the hostname of the Ingress object.
+|type: string
+|
+
+|`ingressClassName`
+|IngressClassName is the name of an IngressClass cluster resource. Ingress controller implementations use this field to know whether they should be serving this Ingress resource.
```

Comment:
```suggestion
|IngressClassName is the name of an IngressClass cluster resource. This field defines whether Ingress controller implementations are to serve this Ingress resource.
```

Reply:
```suggestion
|IngressClassName is the name of an IngressClass cluster resource. This field defines which Ingress controller serves this Ingress resource.
```

There can be multiple ingress controllers in the cluster, and this field defines which of these should handle this ingress.

Reply:
Thank you. Applying.